ReactNativeClient/lib/components/note-body-viewer.js

```diff
@@ -203,6 +210,24 @@ class NoteBodyViewer extends Component {
 		return this.forceUpdate_;
 	}
 
+	async onResourceLongPress(resourceId) {
+		const resource = await Resource.load(resourceId);
+		const name = resource.title ? resource.title : resource.file_name;
+
+		const action = await dialogs.pop(this, name, [
+			{ text: _('Share'), id: 'share' },
```

Comment:
So far it's only share. I thought about adding a separate export item but it seems that share already includes that, at least on my device.
Maybe worth adding a properties screen showing id, created/updated dates, type, size, file name, etc?

Reply:
Would it be difficult to add an "Open" item, which would simply open the resource like it does now when you do a simple press? I think this is what's normally expected, to have the default action on top of the menu.